css/style.css

```
@@ -92,49 +106,91 @@ body {
   height: 100vh;
   background: url(../img/intro2.jpg) no-repeat center;
   background-size: cover;
-  padding-top: 350px;
+  padding-top: 240px;
 }
 
 .intro_title {
   font-weight: 900;
   font-size: 30px;
   line-height: 36px;
-  color: black;
   max-width: 700px;
   margin-bottom: 100px;
 }
 .intro_subtitle {
   font-weight: 700;
   font-size: 24px;
   line-height: 36px;
-  color: black;
   max-width: 700px;
   margin-bottom: 30px;
 }
 .help_wrap {
   font-weight: 700;
   font-size: 20px;
   line-height: 24px;
-  color: black;
   max-width: 500px;
+  padding-bottom: 100px;
+  padding-top: 14px;
+  border: 2px solid #e1a334;
+  border-radius: 10px;
 }
-.result {
-  font-weight: 500;
+
+.fieldText {
+  background-color: #fee6a6;
+  border-radius: 5px;
+}
+
+.fieldText:focus {
```

кстати, ниже есть аналогичный стиль, почему не использовать его два раза?

```suggestion
.fieldText:focus, .btСalculation:focus {
```